test/test_csg.jl

```diff
@@ -1,4 +1,5 @@
 @testset verbose=true "CFGs" begin
```

Comment:
This currently tests only cfgs (as to be expected with the name change), but maybe we should add some csg tests too then?

Reply:
Already renamed that one! :)